### ISSUE TITLE
Get rid of maven-compat

### ISSUE THREAD

**Issue by laeubi** [eclipse/tycho#1138]
We have a lots of `maven-compat` references and this is APi that will be gone with maven 4 so we should start migration see https://github.com/apache/maven/pull/765

**Comment by laeubi** [comment]
It seems I was too optimistic. Now the test execution failed, I tried to make maven-core test-scoped but then compilation failed ???

**Comment by akurtakov** [comment]
Yeah, I've been removing maven-compat bit by bit for the last few years. It's never that simple :). I could only recommend doing it in smaller chunks so you can have some progress at least.
